jboss/container/util/pkg-update/execute.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+for mgr in yum dnf microdnf; do
+    if test -x "/usr/bin/$mgr"; then
```

Comment:
I'd suggest to use `command -v "$mgr"` here over `test -x` with a path. This avoids hard-coding `/usr/bin`